### PR TITLE
Pass step to web component

### PR DESCRIPTION
**Pull request by loiswells97**

### CODE REVIEW

File: src/events/WebComponentCustomEvents.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const webComponentCustomEvent = (type, detail) =>
+  new CustomEvent(type, {
+    bubbles: true,
+    cancelable: false,
+    composed: true,
```
Comment (PetarSimonovic) [inline src/events/WebComponentCustomEvents.js:5]: Would we ever need to set other values for `bubbles`, `cancelable` or `composed`? Could we provide defaults but allow overrides? Or is that unnecessary?

**Review by PetarSimonovic — COMMENTED** [review]
Just one query Lois

File: src/events/WebComponentCustomEvents.js (hunk 1 above)
Comment (loiswells97) [reply src/events/WebComponentCustomEvents.js:5]: Not sure tbh, these are all we're using for now. I guess if we do need other values in the future we can generalise it a bit more?

**Review by PetarSimonovic — APPROVED** [review]
lgtm 👍

**Pull request merged**